Makefile

```diff
@@ -9,23 +9,28 @@ export ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
 GOLANG_VERSION := $(shell sed -En 's/^go (.*)$$/\1/p' "go.mod")
 # Image URL to use all building/pushing image targets
-ifeq ($(origin IMAGE_REPO), undefined)
-IMAGE_REPO := quay.io/operator-framework/operator-controller
+ifeq ($(origin IMG_NAMESPACE), undefined)
+IMG_NAMESPACE := quay.io/operator-framework
```

Comment:
I would prefer REGISTRY
instead since we publish the image into the registry
but I do not think that it is a blocker for this one.

Reply:
I like REGISTRY better, but we can update later

Reply:
+1 agree with @camilamacedo86